.vscode/settings.json

```diff
@@ -18,4 +18,5 @@
     ],
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
+    "restructuredtext.pythonRecommendation.disabled": true,
```

Comment:
what does this do?

Reply:
not sure anymore, can probably be removed...